src/askscript/__tests__/15-more-resources/more-resources-01-jsonPath.ask

```diff
@@ -0,0 +1,12 @@
+ask {
+    let revPerClient = {
+        A: 136,
+        B: 426,
+        C: 133,
+        D: 35,
+        E: 246,
+        F: 446,
+        G: 53,
+    }
+    revPerClient:jsonPath('$.A')
```

Comment:
Would be great to provide more examples and return them as dictionary, thoughts?

Reply:
Done!